Sources/Network/Parse.swift

```diff
@@ -2,15 +2,17 @@
 //  Parse.swift
 //  Alicerce
 //
-//  Created by Luís Afonso on 06/04/2017.
+//  Created by Luís Portela on 06/04/2017.
 //  Copyright © 2017 Mindera. All rights reserved.
 //
 
 import UIKit
 
+public protocol ParseError: Swift.Error {}
```

Comment:
We could have some extra type safety by forcing all errors coming out of the parse step to conform to this protocol, but then it doesn't model any specific behavior so I don't see a clear advantage.